compiler/resolution/lowerForalls.cpp

```diff
@@ -887,9 +887,8 @@ static void expandTaskFn(ExpandVisitor* EV, CallExpr* callToTFn, FnSymbol* taskF
   if (addErrorArgToCall)
     addDummyErrorArgumentToCall(callToTFn);
 
-  // If we don't flatten it right away, we get non-global taskFns
-  // in expandTaskFn(). That may cause issues with scoping.
   flattenNestedFunction(cloneTaskFn);
+
```

Comment:
What is the reason for removing the comment?

The addition of an empty line seems unwarranted.

Reply:
This is a mistake, I will revert it. Thank you!